chapters/Beacons.md

```diff
@@ -0,0 +1,725 @@
+::Beacon Signals:: broadcast from this ::Beacon:: MUST be processed as part of
+resolution. The type of the ::Beacon:: service in the DID document defines how
+::Beacon Signals:: MUST be processed.
+
+**did:btc1** supports different ::Beacon Types::, with each type defining a set
+of algorithms for:
+
+1. How a ::Beacon:: can be established and added as a service to a DID document.
+1. How attestations to DID updates are broadcast within ::Beacon Signals::.
+1. How a resolver processes a ::Beacon Signal::, identifying, verifying, and
+   applying the authorized mutations to a DID document for a specific DID.
+
+This is an extensible mechanism, such that in the future new ::Beacon Types::
+could be added.
+
+The current, active ::Beacons:: of a DID document are specified in the document's
+`service` property. By updating the DID document, a DID controller can change
+the set of ::Beacons:: they use to broadcast updates to their DID document over
+time. Resolution of a DID MUST process signals from all ::Beacons:: identified in the
+latest DID document and apply them in the order determined by the version specified
+by the `didUpdatePayload`.
+
+All resolvers of **did:btc1** DIDs MUST support the Beacon Types defined in this
+specification.
+
+The commitment is a SHA256 hash that represents one of the following:
+
+* the hash of a single DID update payload associated with a **did:btc1**
+  identifier;
+* the hash of a document of key-value pairs, where each key is a **did:btc1**
+  identifier and the value is the hash of a DID update payload or null for that
+  key; or
+* the hash of a [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree) root,
+  where each leaf node is deterministically selected by a **did:btc1**
+  identifier and contains a hash associated with the **did:btc1** identifier.
+
+When defining a service for a ::Beacon:::
+
+* `type` is one of "SingletonBeacon", "MapBeacon", or "SMTBeacon"
+* `serviceEndpoint` is a Bitcoin address represented as a URI following the
+  [BIP21 scheme](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki)
+
+How the Bitcoin address and the cryptographic material that controls it are
+generated is left to the implementation.
+
+### Actors
+
+Actors in signaling are as follows:
+
+* DID controller - a party that controls one or more **did:btc1** identifiers
+  participating in a ::Beacon::
+* cohort - a set of DID controllers participating in a ::Beacon:: and in the
+  construction of the n-of-n P2TR Bitcoin address for the ::Beacon::
+* aggregator - the party responsible for aggregating updates from all DID
+  controllers and creating the ::Beacon Signals::
+* verifier - a party verifying a **did:btc1** identifier presentation
+
+### Aggregation
+
+Three types of ::Beacons:: are defined, with two of them aggregating multiple
+DID update payloads into a single transaction. How coordination between an
+aggregator and multiple DID controllers is managed is out of scope, but one
```

Comment:
```suggestion
Three types of ::Beacons:: are defined: SingletonBeacon, MapBeacon and SMTBeacon. 
Two of them, MapBeacon and SMTBeacon, support aggregation, i.e. the act of 
committing to multiple DID update payloads into a single transaction.
```

@jintekc I would encourage you to submit change requests like this. I like your suggestion, I tweaked it slightly (combining to committing)